src/components/taskDetails/TaskDates.tsx

```diff
@@ -32,20 +32,23 @@ export const TaskDates: React.FC<TaskDatesProps> = ({
     return (
         <>
             <div className={styles.inputContainer}>
-                <Details detailType="Started On" value={startedOn} />
+                <Details
+                    detailType="startedOn"
+                    value={'3/30/2024, 11:20:00 AM'}
```

Comment:
Why hardcode this @iAmAshuSahoo ?

Reply:
I am now maintaining the values in a constant file.

Reply:
Why is this date still hardcoded? @iAmAshuSahoo

Reply:
As a part of this task, we needed to move enums in the constant file. Other values are used as BAU.

Reply:
I have fixed that.